### ISSUE TITLE
"Accessibility Help" dialog while using screen reader

### ISSUE THREAD

**Issue by galyech** [Microsoft/monaco-editor#506]
monaco-editor version: 0.9.0
Browser: Edge  14.14393
OS: Windows 10

Steps or JS usage snippet reproducing the issue:
Accessibility issue: when the Accessibility Help dialog is opening, the screen reader says:
 **"Dialog focused on: Null"** 
instead of : "Dialog has opened" / "Accessibility help dialog has opened" Etc. 

Furthermore, it doesn't focused on the text. i.e it does not read the text inside the dialog and the text is not navigable, only with "item" mode of the screen reader. 

The expected result is, focus should be given to the text in the dialog. 

Thanks.

**Comment by rebornix** [comment]
@galyech thanks for reporting this issue!
